pypit/arqa.py

```diff
@@ -670,24 +670,24 @@ def slit_profile(slf, mstrace, model, lordloc, rordloc, msordloc, textplt="Slit"
                 continue
             spatval = (word[1] + 0.5 - lordloc[:, ndone+j][word[0]]) / (rordloc[:, ndone+j][word[0]] - lordloc[:, ndone+j][word[0]])
             fluxval = mstrace[word]
-            mxval = np.max(fluxval)
+            mxval = 1.25
```

Comment:
I'll trust that this is a good idea.

Reply:
Yep - on line 644 the maximum value is 1.25 (the slit profile is normalized to 1, so this is just a bit above 1).